guard/src/commands/validate.rs

```diff
@@ -436,6 +439,17 @@ or rules files.
     }
 }
 
+pub(crate) fn validate_path_buf(base: &str) -> Result<()> {
+    let file_or_dir = Path::new(base);
+    if !file_or_dir.is_file() && !file_or_dir.is_dir() {
```

Comment:
Is this more optimal than the following? Since we can check without breaking it down.

```
use std::path::Path;

if !Path::new(file_or_dir).exists() {

```

Reply:
good suggestion, definitely cleaner. 

I just stumbled upon this, which also checks for permissions

https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.try_exists